python/lsst/ts/externalscripts/maintel/make_comcam_bias.py

```diff
@@ -0,0 +1,163 @@
+# This file is part of ts_externalscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["MakeComCamBias"]
+
+import json
+import yaml
+import os
+
+from ..base_make_bias import BaseMakeBias
+from lsst.ts import salobj
+from lsst.ts.observatory.control.maintel.comcam import ComCam
+
+
+class MakeComCamBias(BaseMakeBias):
+    """ Take biases and construct a master bias SAL Script.
+
+    This class takes biases with LSSTComCam and constructs
+    a master bias calling the bias pipetask via OCPS.
+    """
+
+    def __init__(self, index=1):
+        super().__init__(
+            index=index,
+            descr="This class takes biases with LSSTComCam and constructs "
+                  "a master bias calling the bias pipetask via OCPS.",
+        )
+        self.comcam = ComCam(domain=self.domain, log=self.log)
+        self.ocps = salobj.Remote(domain=self.domain, name="OCPS")
+
+    @classmethod
+    def get_schema(cls):
```

Comment:
I don't think you need to override this method from the parent class. If you really want to override it because of the tule and description, make sure you do:

```
        schema_dict = yaml.safe_load(schema)

        base_schema_dict = super(TakeImageComCam, cls).get_schema()

        for prop in base_schema_dict["properties"]:
            schema_dict["properties"][prop] = base_schema_dict["properties"][prop]

        return schema_dict
```